tests/storage/export.go

```diff
@@ -572,8 +573,22 @@ var _ = Describe(SIG("Export", func() {
 		}
 		By(fmt.Sprintf("Downloading from URL: %s", downloadUrl))
 		Eventually(ThisPod(downloadPod), 30*time.Second, 1*time.Second).Should(HaveConditionTrue(k8sv1.PodReady))
-		out, stderr, err := exec.ExecuteCommandOnPodWithResults(downloadPod, downloadPod.Spec.Containers[0].Name, command)
-		Expect(err).ToNot(HaveOccurred(), "out: %s stderr: %s", out, stderr)
+		var out, stderr string
+		// Eventual consistency will bail us out in the unlikely event that the proxy certs rotate mid test
+		// And take some time to reload in the download pod
+		var regenOnce sync.Once
+		Eventually(func() error {
+			out, stderr, err = exec.ExecuteCommandOnPodWithResults(downloadPod, downloadPod.Spec.Containers[0].Name, command)
+			if err != nil && strings.Contains(stderr, "failed to verify the legitimacy of the server") {
```

Comment:
Do we expect this message to be stable or would it be better to check a return code if possible?

Reply:
I think we can just skip the message check, and just try it on any error.